ios/COVIDSafePaths.xcodeproj/xcshareddata/xcschemes/BT_Development.xcscheme

```diff
@@ -111,7 +111,7 @@
       buildConfiguration = "Debug-BT">
    </AnalyzeAction>
    <ArchiveAction
-      buildConfiguration = "Release-BT"
+      buildConfiguration = "Debug-BT"
```

Comment:
This is probably me talking from lack of knowledge but won't this build the schema that needs the metro server?

Reply:
good question - since it's archiving it will not be an issue 👍